src/peilbeheerst_model/01_parse_crossings.py

```diff
@@ -25,8 +25,9 @@
 for waterschap, waterschap_struct in waterschap_data.items():
     print(f"\n{waterschap}...")
 
-    init_settings, crossing_settings = waterschap_struct.values()
-    init_settings["logfile"] = pathlib.Path(init_settings["output_path"]).with_suffix("").with_suffix(".log")
+    crossing_settings = waterschap_struct["find_crossings_with_peilgebieden"]
+    init_settings = waterschap_struct["init"]
+    init_settings["logfile"] = pathlib.Path(init_settings["output_path"]).with_suffix(".log")
```

Comment:
The order of the `waterschap_struct.values()` changed, so this was no longer correct.
And `with_suffix` replaces any existing suffix, so a single call is sufficient.